src/CidrValidator.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace ipl\Validator;
+
+use ipl\I18n\Translation;
+use ipl\Stdlib\Str;
+
+/**
+ * Validate a classless inter-domain routing (CIDR)
+ */
+class CidrValidator extends BaseValidator
+{
+    use Translation;
+
+    public function isValid($value): bool
+    {
+        $this->clearMessages();
+
+        $pieces = Str::trimSplit($value, '/');
+        if (count($pieces) !== 2) {
+            $this->addMessage(sprintf($this->translate('CIDR "%s" is in the wrong format'), $value));
```

Comment:
Is it necessary to include the wrong value since our form discards it upon rerendering the form?

Reply:
I don't know what exactly you mean by this. I included the wrong CIDR in the message because you can configure comma-separated CIDRs in `JobConfigForm` and thus immediately see which CIDR failed validation. And as far as I know, the form doesn't discard any values.

Reply:
How does it validate multiple CIDRs?

Reply:
Just like this:
https://github.com/Icinga/icingaweb2-module-reporting/blob/a801bf9f70ab2bc56fd9583c0b7b82280748be9c/library/Reporting/Actions/SendMail.php#L89-L103

Reply:
Check. Thanks.